apps/web/pages/api/book/event.ts

```diff
@@ -223,7 +230,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const reqBody = req.body as BookingCreateBody;
 
   // handle dynamic user
-  const dynamicUserList = getUsernameList(reqBody?.user);
+  const dynamicUserList = Array.isArray(reqBody.user)
+    ? getGroupName(req.body.user)
+    : getUsernameList(reqBody.user as string);
```

Comment:
This change was recently introduced, and I was getting error when booking a collective/round robin event due to multiple users being sent [user1, user2]